package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "movieapp",
+  "version": "1.0.0",
+  "description": "movie application",
+  "main": "index.js",
+  "scripts": {
+    "dev": "webpack-dev-server --config webpack.dev.js --env.development",
+    "build": "webpack --config webpack.prod.js --env.production",
+    "test": "jest"
+  },
+  "keywords": [
+    "movie",
+    "mark"
+  ],
+  "author": "Mykhailo",
+  "license": "ISC",
+  "dependencies": {
+    "babel-jest": "^24.5.0",
+    "react-redux": "^6.0.1",
+    "react-router": "^4.3.1",
+    "react-test-renderer": "^16.8.4",
```

Comment:
`babel-jest` and `react-test-renderer` are dev dependencies

Reply:
fixed